### PR TITLE
Adding cropping for gaussian splat exports

### PR DESCRIPTION
**Pull request by THE-COB**
- Enabled obb in the viewer command generation
- Added obb support in the export

### CODE REVIEW

File: nerfstudio/scripts/exporter.py [hunk 1]
```diff
@@ -585,6 +599,37 @@ def main(self) -> None:
...
+            ):
+                crop_obb = OrientedBox.from_params(self.obb_center, self.obb_rotation, self.obb_scale)
+                assert crop_obb is not None
+                mask = crop_obb.within(torch.from_numpy(positions)).numpy()
+                for k, t in map_to_tensors.items():
+                    map_to_tensors[k] = map_to_tensors[k][mask]
+
+                n = map_to_tensors["x"].shape[0]
+                count = n
+            elif self.use_bounding_box:
```
Comment (kerrj) [inline nerfstudio/scripts/exporter.py:616]: same here, can get rid of the old version

**Comment by kerrj** [comment]
solves #3042

File: nerfstudio/scripts/exporter.py [hunk 2]
```diff
@@ -491,6 +491,20 @@ class ExportGaussianSplat(Exporter):
     Export 3D Gaussian Splatting model to a .ply
     """
 
+    use_bounding_box: Optional[bool] = False
+    """Only query points within the bounding box"""
+    bounding_box_min: Optional[Tuple[float, float, float]] = (-1, -1, -1)
```
[the remark this answers is not in the document]
Comment (THE-COB) [reply nerfstudio/scripts/exporter.py:496]: should we still keep the use_bounding_box parameter or just check if the obb is None?

[the remark this answers is not in the document]
Comment (kerrj) [reply nerfstudio/scripts/exporter.py:496]: I think having the boolean is useful for toggling is more easily from the command line so maybe let's keep it. I'd be in favor of removing the old min/max code from the other export too, @brentyi what do you think?

[the remark this answers is not in the document]
Comment (THE-COB) [reply nerfstudio/scripts/exporter.py:496]: Makes sense, I'll do that tomorrow

[the remark this answers is not in the document]
Comment (THE-COB) [reply nerfstudio/scripts/exporter.py]: I believe I have done it

**Review by kerrj — APPROVED** [review]
lgtm thanks!

**Pull request merged**